src/wazuh_mcp_server/utils/pydantic_compat.py

```
@@ -0,0 +1,132 @@
+
+except ImportError as e:
+    # Pydantic not available
+    pydantic_available = False
+    PYDANTIC_V2 = False
+    
+    # Create minimal fallback classes
+    class BaseModel:
+        def __init__(self, **kwargs):
+            for key, value in kwargs.items():
+                setattr(self, key, value)
+    
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)